### PR TITLE
pid request for cupkee

### PR DESCRIPTION
**Pull request by cupkee**

### CODE REVIEW

**Comment by Arachnid** [comment]
Thanks for the Pull request!

Please follow the guide at http://pid.codes/howto; specifically, you need to create an org file that matches the description there. If your project requires custom hardware, the designs must also be checked into your source repository under an open-source license.

**Comment by cupkee** [comment]
I'll fix it, thanks

**Comment by cupkee** [comment]
I moved my index.md to org/cupkee.

**Comment by peternewman** [comment]
Hi @cupkee 

As per the notes on http://pid.codes/howto/ you need to split your file up, you need a file in 1209/2016/index.md with the layout: pid stuff and another file in org/cupkee/index.md with the layout: org stuff.

**Comment by cupkee** [comment]
I really should read the instruction manual carefully

File: 1209/2016/index.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+layout: pid
+title: Cupkee
+owner: cupkee
+license: MIT
+site: http://www.cupkee.cn/
+source: https://github.com/cupkee/cupkee/
+---
+Cupkee is a operating system, it provides a node-like envrionment on mirco-controller.
```
Comment (peternewman) [inline 1209/2016/index.md]: Nit, SPaG, micro-controller

Comment (peternewman) [inline 1209/2016/index.md]: Nit, SPaG, environment

Comment (peternewman) [inline 1209/2016/index.md]: Nit, SPaG, "is an operating system"

File: org/cupkee/index.md [hunk 2]
```diff
@@ -1,16 +1,9 @@
 ---
-layout: pid
-title: Cupkee
-owner: Cupkee
-license: MIT
-site: http://www.cupkee.cn
-source: https://github.com/cupkee/cupkee
+layout: org
+title: Lixing Ding
```
Comment (peternewman) [inline org/cupkee/index.md]: This probably wants to be Cupkee too. Otherwise you probably want to put it in a lixingding org folder instead.

**Comment by cupkee** [comment]
hi @peternewman 

Thank you for your meticulous proofreading.
I adjusted immediately.

**Pull request merged**